### PR TITLE
fix(registration-react): add default route handler

### PR DESCRIPTION
**Pull request by markkopaas**

### CODE REVIEW

File: applications/registration-react/src/app/protected-route.component.jsx [hunk 1]
```diff
@@ -24,10 +29,11 @@ export const ProtectedRoute = ({ check, ...props }) => {
 };
 
 ProtectedRoute.propTypes = {
-  check: PropTypes.func.isRequired,
+  check: PropTypes.func,
   computedMatch: PropTypes.object
 };
 
 ProtectedRoute.defaultProps = {
+  check: T,
```
Comment (azaxarov) [inline applications/registration-react/src/app/protected-route.component.jsx]: ??

File: applications/registration-react/src/services/auth/login-store.js [hunk 2]
```diff
@@ -1,14 +1,24 @@
...
+export const setLoginState = value => {
+  const valueString = JSON.stringify({ ...value });
+  sessionStorage.setItem('login', valueString);
 };
 
 export const clearLoginState = () => {
   sessionStorage.removeItem('login');
 };
+
+export const popLoginState = () => {
```
Comment (azaxarov) [inline applications/registration-react/src/services/auth/login-store.js:20]: Where is this function used?

File: applications/registration-react/src/services/auth/auth-service.js [hunk 3]
```diff
@@ -40,13 +39,18 @@ function initialize() {
...
 export async function initAuthService() {
   const kcConfig = getConfig().keycloak;
   kc = Keycloak(kcConfig);
 
   setOnAuthSuccess();
 
-  await initialize();
+  await initializeKeycloak();
+  if (isAuthenticated()) {
+    clearLoginState();
```
Comment (azaxarov) [inline applications/registration-react/src/services/auth/auth-service.js]: Use pop here?

**Review by azaxarov — APPROVED** [review]

File: applications/registration-react/src/services/auth/auth-service.js (hunk 3 above)
Comment (markkopaas) [reply applications/registration-react/src/services/auth/auth-service.js]: takk, fikset

File: applications/registration-react/src/app/protected-route.component.jsx (hunk 1 above)
Comment (markkopaas) [reply applications/registration-react/src/app/protected-route.component.jsx]: takk, fikset

**Pull request merged**